tests/test_status_change.py

```diff
@@ -488,9 +521,23 @@ async def test_ci_passed_and_check_run_timed_out_awaiting_merge_label_pr_is_not_
         },
     }
 
-    gh = FakeGH(getitem=getitem)
+    getiter = {
+        "/repos/python/cpython/pulls/5547/commits": [
+            {
+                "sha": "f2393593c99dd2d3ab8bfab6fcc5ddee540518a9",
+                "commit": {
+                    "message": "bpo-32720: Fixed the replacement field grammar documentation. (GH-5544)\n\n`arg_name` and `element_index` are defined as `digit`+ instead of `integer`.\n(cherry picked from commit 7a561afd2c79f63a6008843b83733911d07f0119)\n\nCo-authored-by: Mariatta <Mariatta@users.noreply.github.com>"
+                },
+            }
+        ]
+    }
+
+    gh = FakeGH(getitem=getitem, getiter=getiter)
     await status_change.router.dispatch(event, gh)
-    assert len(gh.post_data["body"]) is not None  # leaves a comment
+    assert (
+        gh.post_data["body"]
+        == "@miss-islington and @Mariatta: Status check is done, and it's a failure or timed out ❌."
+    )
```

Comment:
In order to avoid awkward indentation, long lines, and to make the code shorter and more readable, I would suggest the following reformatting:
```suggestion
    expected_body = ("@miss-islington and @Mariatta: Status check is done, "
                      "and it's a failure or timed out ❌.")
    assert gh.post_data["body"] == expected_body
```